src/kem/kyber/CMakeLists.txt

```diff
@@ -1,97 +1,142 @@
 # SPDX-License-Identifier: MIT
 
 # This file was generated by
-# scripts/copy_from_pqclean/copy_from_pqclean.py
+# scripts/copy_from_upstream/copy_from_upstream.py
 
 set(_KYBER_OBJS "")
 
+if(OQS_ENABLE_KEM_kyber_512 OR OQS_ENABLE_KEM_kyber_1024 OR OQS_ENABLE_KEM_kyber_512_90s OR OQS_ENABLE_KEM_kyber_768_90s OR OQS_ENABLE_KEM_kyber_1024_90s)
+    add_library(kyber_common_ref OBJECT pqcrystals-kyber_common_ref/aes256ctr.c pqcrystals-kyber_common_ref/fips202.c)
+    target_include_directories(kyber_common_ref PRIVATE ${CMAKE_CURRENT_LIST_DIR}/pqcrystals-kyber_common_ref)
+    set(_KYBER_OBJS ${_KYBER_OBJS} $<TARGET_OBJECTS:kyber_common_ref>)
+endif()
+
+if(OQS_ENABLE_KEM_kyber_512_avx2 OR OQS_ENABLE_KEM_kyber_768_avx2 OR OQS_ENABLE_KEM_kyber_1024_avx2 OR OQS_ENABLE_KEM_kyber_512_90s_avx2 OR OQS_ENABLE_KEM_kyber_768_90s_avx2 OR OQS_ENABLE_KEM_kyber_1024_90s_avx2)
+    add_library(kyber_common_avx2 OBJECT pqcrystals-kyber_common_avx2/fips202.c pqcrystals-kyber_common_avx2/fips202x4.c)
+    target_include_directories(kyber_common_avx2 PRIVATE ${CMAKE_CURRENT_LIST_DIR}/pqcrystals-kyber_common_avx2)
+    target_compile_options(kyber_common_avx2 PRIVATE -mavx2)
+    set(_KYBER_OBJS ${_KYBER_OBJS} $<TARGET_OBJECTS:kyber_common_avx2>)
+endif()
+
+if(OQS_ENABLE_KEM_kyber_512_avx2 OR OQS_ENABLE_KEM_kyber_768_avx2 OR OQS_ENABLE_KEM_kyber_1024_avx2)
+    add_library(kyber_common_keccak4x_avx2 OBJECT pqcrystals-kyber_common_keccak4x_avx2/keccak4x/KeccakP-1600-times4-SIMD256.c)
+    target_include_directories(kyber_common_keccak4x_avx2 PRIVATE ${CMAKE_CURRENT_LIST_DIR}/pqcrystals-kyber_common_keccak4x_avx2)
+    target_compile_options(kyber_common_keccak4x_avx2 PRIVATE -mavx2)
+    set(_KYBER_OBJS ${_KYBER_OBJS} $<TARGET_OBJECTS:kyber_common_keccak4x_avx2>)
+endif()
+
+if(OQS_ENABLE_KEM_kyber_512_90s_avx2 OR OQS_ENABLE_KEM_kyber_768_90s_avx2 OR OQS_ENABLE_KEM_kyber_1024_90s_avx2)
+    add_library(kyber_common_aes OBJECT pqcrystals-kyber_common_aes/aes256ctr.c)
+    target_include_directories(kyber_common_aes PRIVATE ${CMAKE_CURRENT_LIST_DIR}/pqcrystals-kyber_common_aes)
+    target_compile_options(kyber_common_aes PRIVATE -maes -msse2 -mssse3)
```

Comment:
Ahh -- this explains my confusion probably: Does SSSE3 and SSE2 only apply to Kyber90s but not Kyber (-noAES)? If so, let's go ahead and merge.

Reply:
Yes,  SSSE3 and SSE2 is part of the AES-dependency, so only relevant for Kyber90s.